### PR TITLE
Add LogDNA CLI v1.0.9

### PR DESCRIPTION
**Pull request by beefcheeks**
#### Adding a new cask
- [x] Checked there aren’t open [pull requests](https://github.com/caskroom/homebrew-cask/pulls) for the same cask.
- [x] Checked there aren’t closed [issues](https://github.com/caskroom/homebrew-cask/issues) where that cask was already refused.
- [x] When naming the cask, followed the [token reference](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] Commit message includes cask’s name.
- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` left no offenses.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.

### CODE REVIEW

File: Casks/logdna-cli.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask 'logdna-cli' do
+  version '1.0.9'
+  sha256 '76531e0538c9c8945ade50fc5707e1d34945b364a6475f38af6ccd499aad2aaa'
+
+  url 'https://github.com/logdna/logdna-cli/releases/download/1.0.9/logdna-mac-cli.pkg'
```
Comment (vitorgalvao) [inline Casks/logdna-cli.rb]: `url "https://github.com/logdna/logdna-cli/releases/download/#{version}/logdna-mac-cli.pkg"`

Comment (vitorgalvao) [inline Casks/logdna-cli.rb]: Please add an [`appcast`](https://github.com/caskroom/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

Comment (baronomasia) [inline Casks/logdna-cli.rb]: Fixed.

Comment (vitorgalvao) [inline Casks/logdna-cli.rb]: Thank you for adding the `appcast`, but you forgot to change `url`.

**Comment by baronomasia** [comment]
@vitorgalvao Added the app cast and re-ran all the checks. Let me know if it looks good.

File: Casks/logdna-cli.rb (hunk 1 above)
Comment (baronomasia) [inline Casks/logdna-cli.rb]: Whoops you're right - I've fixed that now. Had some difficulty and forgot that single quotes don't evaluate variables (e.g. '#{version}' doesn't work). Let me know if this looks good.

Comment (vitorgalvao) [inline Casks/logdna-cli.rb]: Looking great. Thank you. Will merge after Travis passes.

**Pull request merged**